src/ggml-vulkan.cpp

```diff
@@ -1,3 +1,6 @@
+// SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+// SPDX-License-Identifier: MIT
+
```

Comment:
Why this note? I don't think this is appropriate. If we put a copyright note here, it'd have to mention others before Nvidia.

Reply:
It's company policy to add a copyright notice in files that are modified. I'm happy to add other notes here as well, maybe `Copyright (c) 2023-2024 The ggml authors` like is in some other files?

Reply:
I'm the main author of this file, but many people have contributed. Should we mention them all? The more significant ones? Or just a group?

@ggerganov Do you have an opinion here?

Reply:
> @ggerganov Do you have an opinion here?

I don't really understand the necessity for adding these SPDX identifiers in the source files. The copyright and license of the project is already specified in the [LICENSE](https://github.com/ggerganov/ggml/blob/master/LICENSE) file, so these identifiers are IMO pointless.

> Should we mention them all? The more significant ones? Or just a group?

We definitely don't want to maintain redundant copyright notices and license identifiers in the source files. There is commit history already available which provides all that information in greater detail. Any entity that wants to claim copyright should be added to the [AUTHORS](https://github.com/ggerganov/ggml/blob/master/AUTHORS) list - this is the main purpose for the file to exist and it's much easier to maintain.

IMO the simplest solution is to remove the identifiers.

Reply:
OK, I think the AUTHORS file will suffice. I've regenerated that file and removed the copyright comments.